project/api/models/thread.py

```diff
@@ -28,6 +28,8 @@ def summarize(self, thread):
             "summary": thread.summary[:thread_truncate_length] + (ellipsis_if_too_long),
             "created": "{0} {1}, {2}".format(month_name[thread.created.month], thread.created.day, thread.created.year),
             "category_id": thread.category.id,
+            "state": thread.state if thread.level == "state" else "federal",
```

Comment:
Just a thought, but could `state` allow Null values?

Reply:
Sorry, the 'state' property isn't being used (it was a false start on my part). I'm using the location parameter instead. I've updated the code to remove the state paramter.

The location parameter does handle having a null state. It'll just print "state". Hope this helps. Thanks.